lib/features/announcement/ui/announcements_list/setting_page.dart

```diff
@@ -14,32 +15,271 @@ class _VocelSetting extends State<VocelSetting>{
+                  boxShadow: [
+                    BoxShadow(
+                      color: Colors.grey.withOpacity(0.3),
+                      blurRadius: 10,
+                      offset: const Offset(0, 5),
+                    ),
+                  ],
+                ),
+                child: Row(
+                  mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                  children: const [
+                    Text(
+                      'Privacy Settings',
+                      style: TextStyle(
+                        fontSize: 16,
+                      ),
+                    ),
+                    Icon(Icons.arrow_forward_ios, size: 16,),
+                  ],
+                ),
+              ),
+            ),
+            InkWell(
+              onTap: () {
+                // Add your custom logic here
+              },
+              child: Container(
+                margin: const EdgeInsets.fromLTRB(15, 15, 15, 0),
+                padding: const EdgeInsets.symmetric(vertical: 15, horizontal: 20),
+                decoration: BoxDecoration(
+                  color: Colors.white,
+                  borderRadius: BorderRadius.circular(8),
+                  boxShadow: [
+                    BoxShadow(
+                      color: Colors.grey.withOpacity(0.3),
+                      blurRadius: 10,
+                      offset: const Offset(0, 5),
+                    ),
+                  ],
+                ),
+                child: Row(
+                  mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                  children: const [
+                    Text(
+                      'Help and Support',
+                      style: TextStyle(
+                        fontSize: 16,
+                      ),
+                    ),
+                    Icon(Icons.arrow_forward_ios, size: 16,),
+                  ],
+                ),
+              ),
+            ),
+            InkWell(
+              onTap: () {
+                // Add your custom logic here
+              },
+              child: Container(
+                margin: const EdgeInsets.fromLTRB(15, 15, 15, 0),
+                padding: const EdgeInsets.symmetric(vertical: 15, horizontal: 20),
+                decoration: BoxDecoration(
+                  color: Colors.white,
+                  borderRadius: BorderRadius.circular(8),
+                  boxShadow: [
+                    BoxShadow(
+                      color: Colors.grey.withOpacity(0.3),
+                      blurRadius: 10,
+                      offset: const Offset(0, 5),
+                    ),
+                  ],
+                ),
+                child: Row(
+                  mainAxisAlignment: MainAxisAlignment.spaceBetween,
                   children: const [
-                    Text("Languages"),
-                    SizedBox(
-                      height: 30,
+                    Text(
+                      'Log Out',
```

Comment:
pull from a resource file to keep internationalized

Reply:
done, I use "desconectar" in Spanish standing for disconnect for signOut.